sdk/core/azure-core/src/main/java/com/azure/core/spatial/GeometryPosition.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+/**
+ * Represents a geometric position.
+ */
+public final class GeometryPosition {
```

Comment:
Do we want to make these model types `final`? Users or client libraries may want to extend these types.

Reply:
GeoJSON types are well-known and are expected to not be extended, https://tools.ietf.org/html/rfc7946#section-7.